website/docs/admin_hosts_maya.md

```diff
@@ -172,3 +172,12 @@ Fill in the necessary fields (the optional fields are regex filters)
 - Build your workfile
 
 ![maya build template](assets/maya-build_workfile_from_template.png)
+
+## Explicit Plugins Loading
+You can define which plugins to load on launch of Maya here; `project_settings/maya/explicit_plugins_loading`. This can help improve Maya's launch speed, if you know which plugins are needed.
+
+By default only the required plugins are enabled. You can also add any plugin to the list to enable on launch.
+
+:::note technical
+When enabling this feature, the workfile will be launched post initialization no matter the setting on `project_settings/maya/open_workfile_post_initialization`. This is to avoid any issues with references needing plugins.
+:::
```

Comment:
As far as I'm aware those issues with plug-ins don't exist actually if maya got started with the explicit file argument. It was only in rendering with Deadline due to the fact they run Maya with a custom command which opens the file manually after launch.

As such, I believe the issue with plug-ins initializing too late only exists due to "post initialization" and a `maya.exe "/path/to/workfile.ma"` call doesn't suffer from the same issues.

_Yes, it'd load plug-ins after the scene had opened. But as far as I'm aware in that particular case it auto-loads the plug-ins for a file just fine._

Are we sure post initialization is needed?

Anyway, not a big issue. Just wondering.

Reply:
Will investigate. Think it was just something in my Maya `precautions` telling me to do this.

Reply:
Here's a case I cant get to work "pre initialization":

1. Load GPU cache.
2. Have the GPU cache plugin disabled on launch.

@BigRoy unless you have a solution to this, I'll leave it as needing "post initialization".